src/Orleans.Core/AssemblyLoader/AssemblyLoader.cs

```diff
@@ -124,6 +124,20 @@ internal class AssemblyLoader
             }
         }
 
+        /// <summary>
+        /// Create instance for type T in certain assembly, using its parameterless constructor
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="assemblyName"></param>
+        /// <returns></returns>
+        internal static T CreateInstance<T>(string assemblyName)
```

Comment:
It's odd to put this method here with this name, because it is very similar to LoadAndCreateInstance, except that it uses Activator instead of ActivatorUtilities. So if you are going to generalize this and move it to this class, it shouldn't be ambiguous compared to the others

Reply:
hmm it is more like CreateInstanceWithoutDIContainerUsingParametrerlessConstructor... But this is a confusing name. I think I will just put this method in the LegacyMembershipConfigurator.cs file so that other people won't use it